build.cake

```diff
@@ -559,11 +570,12 @@ Task("libs-native")
 
     RunGradle(root, "build");
 
-    DirectoryPath outputDir = MakeAbsolute((DirectoryPath)"./externals/");
+    string outputDir = "./externals/com.xamarin.google.android.material.extensions/";
     EnsureDirectoryExists(outputDir);
+    CleanDirectories(outputDir);
 
     CopyFileToDirectory($"{root}/extensions-aar/build/outputs/aar/extensions-aar-release.aar", outputDir);
-    Unzip($"{outputDir}/extensions-aar-release.aar", $"{outputDir}/extensions-aar");
+    Unzip($"{outputDir}/extensions-aar-release.aar", outputDir);
```

Comment:
Using a sub-folder so that multiple local builds don't blow up with existing files.